### ISSUE TITLE
Feature request: integration in TinyMCE for Moodle 4.2

### ISSUE THREAD

**Issue by Richardvi** [justinhunt/moodle-filter_generico#45]
I'm really sorry if I post this on the wrong place, I'm not that familiar with Github.

But... Moodle is moving away from ATTO and going to TinyMCE. We will start the school year with Moodle 4.2 and TinyMCE and I already know that many teachers will ask for the Generico button in TinyMCE.

I could not find it on Moodle.org plugins yet: https://moodle.org/plugins/browse.php?list=category&id=82

It should be possible as we have a Sketch button and a Components for Learning button in TinyMCE as extra plugins.

**Comment by Richardvi** [comment]
This information might help: https://moodledev.io/docs/apis/plugintypes/tiny

**Comment by justinhunt** [comment]
Yes will do this. I am currently working on a very similar but bigger TinyMCE plugin. Once that is done this should be almost the same. I can not promise when but hopefully before end of September.

**Comment by QueenslandMoodler** [comment]
Hi Justin. Is this the repository for the TinyMCE version? https://github.com/justinhunt/moodle-tiny_generico

Do you plan for existing Generico templates to function in this new version? 

[This Moodle bug](https://tracker.moodle.org/browse/MDL-67360) is pushing us away from Atto but we are dependent on Generico for key site functionality so are keen to follow progress on your work.

**Comment by justinhunt** [comment]
Yes that is the repo, its just a stub right now, please dont use it. I have finished the other plugin, and just waiting for a window of time to get this done. I am getting a bit of heat about it, and its not a super big job. So it will be soon

**Comment by Richardvi** [comment]
Thanks Justin!

**Comment by justinhunt** [comment]
Hi Richard, the first release version is ready now at the tiny_generico repository.
https://github.com/justinhunt/moodle-tiny_generico

 I will add it to Moodle.org in the next day, it might be a few days to weeks till its approved though.

**Comment by Richardvi** [comment]
Many thanks Justin! I will install it in our next update-run end October.
